src/modules/flow/iio/nodes.c

```diff
@@ -143,6 +143,11 @@ iio_direction_vector_reader_cb(void *data, struct sol_iio_device *device)
     r = sol_iio_read_channel_value(mdata->channel_z, &out.z);
     SOL_INT_CHECK_GOTO(r, < 0, error);
 
+    SOL_DBG("Before mount_caliration: %f-%f-%f", out.x, out.y, out.z);
+
+    // mount correction
+    sol_iio_mount_calibration(device, &out.x, &out.y, &out.z);
```

Comment:
Curiosity: why is applying mount matrix optional? Wouldn't be better to always apply it if available? Or are there use cases to not apply it?
If we could always apply it, we could do it internally, and not create a new API.